themes/classic/_dev/css/components/products.scss

```
@@ -755,6 +755,10 @@
 
   .customization-message {
     margin-top: 20px;
+    .customization-label {
```

```suggestion

    .customization-label {
```
Just a little readability improvement